src/gettingstarted/template/next-steps.md

```diff
@@ -0,0 +1,86 @@
+# Start with a template
+
+## Next steps
+
+In this guide you created a project using the [management console](/administration/web.md) and installed the [Platform.sh CLI](/development/cli.md).
+
+Don't stop now! There are far more features that make Platform.sh profoundly helpful to developers that you have left to explore.
+
+### Import your own code
+
+Templates are great, but configuring your own application to run on Platform.sh is just as easy.
```

Comment:
That... is not actually true. :smile: You have to hand edit YAML files, not just push buttons.  It's still easy, but not just as easy.